UNITTESTS/stubs/NetworkInterfaceDefaults_stub.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2017, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "netsocket/NetworkInterface.h"
+#include "CellularBase.h"
+
+void NetworkInterface::set_default_parameters()
```

Comment:
These two need to be removed from here and use NetworkInterfaceDefaults_stub.cpp